spec/features/user_adds_art_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+feature "User adds street art" do
+  scenario "user visits new art page" do
```

Comment:
This scenario is just checking for a blank form and isn't really necessary. I would delete.